source/browseMode.py

```diff
@@ -1778,7 +1795,10 @@ def doResult(result):
 			obj = container
 		return doResult(False)
 
-	def _get_documentConstantIdentifier(self):
+	documentConstantIdentifier: Optional[str]
+	# Mark documentConstantIdentifier property   for caching during the current core cycle
+	_cache_documentConstantIdentifer = True
+	def _get_documentConstantIdentifier(self) -> Optional[str]:
 		"""Get the constant identifier for this document.

```

Comment:
Given this ID can change, the 'constant' in the name is a bit deceptive. Can we at least point this out in the doc-string?